hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -305,8 +300,8 @@ public void createVolume(String volumeName, VolumeArgs volArgs)
     builder.setVolume(volumeName);
     builder.setAdminName(admin);
     builder.setOwnerName(owner);
-    builder.setQuotaInBytes(quotaInBytes);
-    builder.setQuotaInCounts(quotaInCounts);
+    builder.setQuotaInBytes(getQuotaValue(volArgs.getQuotaInBytes()));
+    builder.setQuotaInCounts(getQuotaValue(volArgs.getQuotaInCounts()));
```

Comment:
Arguments are already checked, why change from use of the variables to `getQuotaValue`?

```suggestion
    builder.setQuotaInBytes(quotaInBytes);
    builder.setQuotaInCounts(quotaInCounts);
```